packages/playground/cli/README.md

```diff
@@ -1,76 +1,182 @@
 # WordPress Playground CLI
 
-A CLI tool for running WordPress playground locally instead of in the browser:
+`@wp-playground/cli` streamlines the process of setting up a local WordPress environment for development and testing. It utilizes WordPress Playground to set up a new WordPress environment seamlessly. As its predecessor `wp-now`, you can switch between PHP and WordPress versions only with a flag.
 
-```shell
-$ bun packages/playground/cli/src/cli.ts server --wp=6.5
-WordPress is running on http://127.0.0.1:9400
+# Table of contents
+
+-   [Requirements](#requirements)
+-   [Quickstart](#quickstart)
+-   [Usage](#usage)
+-   [Working with Blueprints](#working-with-blueprints)
+-   [How can I contribute?](#how-can-i-contribute)
+-
+
+## Requirements
+
+The Playground CLI requires Node.js 20.18 or higher, which is the recommended Long-Term Support (LTS) version. You can download it from the [Node.js website](https://nodejs.org/en/download).
+
+## Quickstart
+
+Running the Playground CLI is as simple as going to your plugin or theme directory and running the following command:
+
+```bash
+cd my-plugin-or-theme-directory
+npx @wp-playground/cli server --auto-mount
 ```
 
-Playground CLI is simple, configurable, and unopinionated. You can set it up
-to your unique WordPress setup. For example, this command would run the documentation
-workflow at https://github.com/adamziel/playground-docs-workflow:
+The flag `--auto-mount` will figure out if the project folder is a plugin or a theme for you. For more advanced mounting options, see the [Mounting Local Directories](#mounting-local-directories) section.
+
+## Usage
+
+You don't have to install `@wp-playground/cli`, you can run it directly with `npx`. This is the recommended way to use the CLI and requires no permanent installation. To run a vanilla WordPress website, you can run the command:
+
+```bash
+npx @wp-playground/cli server
+```
+
+### Choosing a WordPress Version
+
+By default, the CLI will load the latest stable version of WordPress and PHP. To specify your preferred versions, you can use the flag `--wp=<version>` and `--php=<version>`:
```

Comment:
It loads PHP 8.0 by default. The reason being it's noticeably faster than 7.4 (lowest [recommended on wp.org](https://wordpress.org/about/requirements/)) and, when we set that version, WordPress was still struggling with PHP 8.1+ compat. Maybe it's time to bump it, but I don't think we should aim for the most recent PHP as a default – things often break on these new versions and core needs time to catch up.